Trend Micro/trend-micro-apex-one/ingest/parser.yml

```
@@ -0,0 +1,573 @@
+          event.type: ["change"]
+          host.name: "{{parsed_event.message.shost}}"
+          source.ip: "{{parsed_event.message.cs2}}"
+          Trend_Micro.Engine: "{{parsed_event.message.cn2}}"
+
+      - translate:
+        dictionary:
+          0: Unused
+          1: In use
+        mapping:
+          parsed_event.message.cn3: Trend_Micro.Engine_Status
+
+  set_logon_logoff_event:
+    actions:
+      - set:
+          event.kind: event
+          event.category: ["authentication"]
+          event.type: ["info"]
+          process.title: "{{parsed_event.message.deviceFacility}}"
+          host.name: "{{parsed_event.message.shost}}"
+          event.reason: "{{parsed_event.message.msg}}"
+
+  set_network_content_inspection_event:
+    actions:
+      - set:
+          event.kind: event
+          event.category: ["network"]
+          event.type: ["connection"]
+          event.code: "{{parsed_event.message.cat}}"
+          process.executable: "{{parsed_event.message.deviceProcessName}}"
+          source.ip: "{{parsed_event.message.src}}"
+          source.port: "{{parsed_event.message.spt}}"
+          destination.ip: "{{parsed_event.message.dst}}"
+          destination.port: "{{parsed_event.message.dpt}}"
+          Trend_Micro.NCIE_ThreatName: "{{parsed_event.message.NCIE_ThreatName}}"
+          event.action: "{{parsed_event.message.act}}"
+          network.direction: >-
+            {% if parsed_event.message.deviceDirection != 'None' %}
+              {{parsed_event.message.deviceDirection}}
+            {% endif %}
+
+  set_pattern_update_status_event:
+    actions:
+      - set:
+          event.kind: event
+          event.category: ["configuration"]
+          event.type: ["change"]
+          host.name: "{{parsed_event.message.shost}}"
+          host.os.name: "{{parsed_event.message.cs1}}"
+          host.ip: "{{parsed_event.message.cs2}}"
+          Trend_Micro.Connection_Status: "{{parsed_event.message.cn1}}"
+          rule.id: "{{parsed_event.message.cn2}}"
+          Trend_Micro.Pattern_RuleVersion: "{{parsed_event.message.cn5}}"
+          event.reason: "{{parsed_event.message.msg}}"
+
+      - translate:
+        dictionary:
+          0: Up-to-date
+          1: 1 version old
+          2: 2 versions old
+          3: 3 versions old
+          4: 4 versions old
+          5: 5 versions old
+          6: 6 versions old
+          7: 7 or more versions old
+        mapping:
+          parsed_event.message.cn3: Trend_Micro.Pattern_RuleStatus
+
+  set_predictive_machine_learning_event:
+    actions:
+      - set:
+          event.kind: alert
+          event.category: ["malware"]
+          event.type: ["info"]
+          file.name: "{{parsed_event.message.fname}}"
+          file.path: "{{parsed_event.message.filePath}}"
+          file.hash.sha1: "{{parsed_event.message.filehash}}"
+          host.name: "{{parsed_event.message.dhost}}"
+          process.user.name: "{{parsed_event.message.duser}}"
```

My bad. `process.user.name` is not defined in the ECS specification. However, `user.target.name` exists and seems more suitable
```suggestion
          user.target.name: "{{parsed_event.message.duser}}"
```